ihatemoney/forms.py

```diff
@@ -171,6 +184,13 @@ class BillForm(FlaskForm):
     what = StringField(_("What?"), validators=[DataRequired()])
     payer = SelectField(_("Payer"), validators=[DataRequired()], coerce=int)
     amount = CalculatorStringField(_("Amount paid"), validators=[DataRequired()])
+    currency_helper = CurrencyConverter()
+    original_currency = SelectField(
+        _("Currency"),
+        choices=currency_helper.get_currencies(),
+        validators=[Optional()],
+        default="USD",
```

Comment:
… and the default value should be the project one here :)

Reply:
done, but in a slightly different way.